joomla/joomla-cms
# [com_tags] Tagged item link in feed

Pull Request for Issue # .

### Summary of Changes

This corrects the links to items in Tagged Items feed.

### Testing Instructions

Create some tags and apply them to some items.
Open Tagged Items feed. Check the item links.

### Expected result

SEF links like this: `https://community.joomla.org/blogs/community/the-marketing-working-group-mwg-welcomes-new-members.html`.

### Actual result

Links like this: `https://community.joomla.org/?view=article&amp;id=1898:the-marketing-working-group-mwg-welcomes-new-members`.

### Documentation Changes Required
No.
